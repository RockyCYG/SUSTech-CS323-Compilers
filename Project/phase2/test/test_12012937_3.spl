int t()
{
	float i;
	i[0]=1+2;
	cout(i+1);
}
