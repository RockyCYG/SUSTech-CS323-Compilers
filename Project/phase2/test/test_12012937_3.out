Error type 10 at Line 4: indexing on non-array variable
Error type 5 at Line 4: unmatching type on both sides of assignment
Error type 2 at Line 5: undefined function: cout

